src/main/java/NodeView.java

```diff
@@ -0,0 +1,83 @@
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashMap;
+
+public class NodeView extends JComponent {
+
+    private final Graph graph;
+    private final int nodeSize = 20;
+    private int networkRadius = 0;
+    private int viewSize = networkRadius + 300;
```

Comment:
The size is messed up at five or more nodes

Reply:
Like when there's specifically five nodes something looks wrong?

Reply:
Yes. And 8 looks weird too.